<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EqSettingsPage</class>
 <widget class="QDialog" name="EqSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(37, 38, 43);</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#2a2c31&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>EQ 设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>72</y>
     <width>540</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot; #232429&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>72</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#2a2c31&quot;;
alternate-background-color:  &quot;#2a2c31&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/icon_back.png</normaloff>:/img/icon_back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>33</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="startTest">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>580</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>开始调试</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>74</y>
     <width>540</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="dialog" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>960</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#2effffff&quot;;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(37, 38, 43);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>正在调试，请勿关闭程序...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="resultDialog" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>960</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#2effffff&quot;;</string>
   </property>
   <widget class="QLabel" name="bg">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>396</y>
      <width>350</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(37, 38, 43);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="commit">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>515</y>
      <width>327</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="line_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>507</y>
      <width>300</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot; #232429&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="testTips">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>400</y>
      <width>330</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(37, 38, 43);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>调试出错，请确保录音设备连接正常！</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>bg</zorder>
   <zorder>testTips</zorder>
   <zorder>commit</zorder>
   <zorder>line_2</zorder>
  </widget>
  <zorder>line</zorder>
  <zorder>widget</zorder>
  <zorder>title</zorder>
  <zorder>back</zorder>
  <zorder>startTest</zorder>
  <zorder>dialog</zorder>
  <zorder>resultDialog</zorder>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
